### DISCUSSION TITLE
Cara Saya Shutdown Pikiran Sebelum Tidur - Rahmat Agung Julians

### DISCUSSION THREAD

**Discussion by giscus** [rahmatagungj/public-rahmatagungjulians.tech#18] [Comments]
# Cara Saya Shutdown Pikiran Sebelum Tidur - Rahmat Agung Julians

Ritual wajib agar dapat tidur nyenyak dan tidak membawa emosi negatif

https://www.rahmatagungjulians.tech/blog/cara-saya-shutdown-pikiran-sebelum-tidur#membuat-jadwal-untuk-besok

**Comment by apriantoa917** [comment]
Wah mantap mas artikelnya. Journaling berguna banget buat jernihin pikiran (yang banyak disepelein & dianggap norak), baik lisan / tulisan. Komunikasi sama diri sendiri, itu yang bisa buat kita termotivasi dan berkembang. Thanks udh share tulisannya 👍🏻🔥🔥🔥
